qa/multi-cluster-search/src/test/resources/rest-api-spec/test/multi_cluster/10_basic.yml

```diff
@@ -115,6 +115,39 @@
   - match: { aggregations.cluster.buckets.0.key: "local_cluster" }
   - match: { aggregations.cluster.buckets.0.doc_count: 5 }
 
+  # once more, this time with a pipeline agg
+  - do:
```

Comment:
Should we have a similar test for the `rolling-upgrade` module?  Theoretically it should be the same as CCS, but it might also smoke out different issues due to heterogeneous serialization inside the same cluster (instead of funneling through a gateway).

Reply:
It'd be great to have a "mixed cluster CCS" test. I talked that one through with @javanna and we don't have one now and probably don't want to build one just for this.